usr/src/man/man4d/bcm2711_soctemp.4d

```diff
@@ -0,0 +1,30 @@
+.\"
+.\" This file and its contents are supplied under the terms of the
+.\" Common Development and Distribution License ("CDDL"), version 1.0.
+.\" You may only use this file in accordance with the terms of version
+.\" 1.0 of the CDDL.
+.\"
+.\" A full copy of the text of the CDDL should have accompanied this
+.\" source.  A copy of the CDDL is also available via the Internet at
+.\" http://www.illumos.org/license/CDDL.
+.\"
+.\"
+.\" Copyright 2025 OmniOS Community Edition (OmniOSce) Association.
+.\"
+.Dd February 9, 2025
+.Dt BCM2711_SOCTEMP 4D
+.Os
+.Sh NAME
+.Nm bcm2711_soctemp
+.Nd Raspberry Pi 4 SoC temperature sensor driver
+.Sh SYNOPSIS
+.Pa /dev/sensors/temperature/cpu/soc
+.Sh DESCRIPTION
+The
+.Nm
+driver provides
+.Pa /dev/sensors/temperature/cpu/soc ,
+a device which exposes the SoC temperature.
+The granularity is in 1000ths of a \(deC.
+.Sh INTERFACE STABILITY
+Evolving
```

Comment:
"Volatile", I think?  But I'm not sure.

Probably should also Xref the main ksensor manual.

Reply:
done